src/EhsnPlugin/Mappers/DischargeActivityMapper.cs

```diff
@@ -247,7 +247,14 @@ private ManualGaugingDischargeSection CreateMidSectionMeasurement(DischargeActiv
             dischargeSection.VelocityUnitId = Units.VelocityUnitId;
             dischargeSection.DeploymentMethod = GetMappedEnum(_ehsn.InstrumentDeployment?.GeneralInfo?.deployment, KnownMidSectionDeploymentTypes);
 
-            AddPanelMeasurements(dischargeSection);
+            var meters = (_ehsn.MidsecMeas?.DischargeMeasurement?.MmtInitAndSummary?.MetersUsed ?? new Meter[0])
+                .Select(CreateMeterCalibration)
+                .ToList();
+
+            if (meters[0].Equations.Count < 2 && meters[1].Equations.Count < 2)
```

Comment:
This line is going to throw an `IndexOutOfRangeException` when the `?? new Meter[0]` null coalescing expression is used.

It isn't clear to me from the existing XML data I've seen that ever happens in real life.

But the way to guard it would be to check the count of the `meters` list before indexing into it.

```C#
if (meters.Count >= 2 && meters[0].Equations.Count < 2 && meters[1].Equations.Count < 2)
```